core/src/main/java/org/sonarsource/sonarlint/core/websocket/WebSocketService.java

```diff
@@ -212,6 +223,7 @@ private void forgetConnection(String connectionId) {
   private void reopenConnection(String connectionId) {
     closeSocket();
     createConnectionIfNeeded(connectionId);
+    handleEventDispatcher(connectionId);
```

Comment:
I think `reopenConnection` is called only in certain cases, maybe you should call this line from `createConnectionIfNeeded`. To be checked